src/main/java/org/gridsuite/modification/utils/ModificationUtils.java

```diff
@@ -589,25 +578,22 @@ public <T> ReportNode applyAndBuildModificationReport(Consumer<T> setter, Suppli
         T oldValue = getter.get();
         T newValue = modification.applyModification(oldValue);
         setter.accept(newValue);
-        return buildModificationReport(oldValue, newValue, fieldName, 1, TypedValue.INFO_SEVERITY);
+        return buildModificationReport(oldValue, newValue, fieldName, TypedValue.INFO_SEVERITY);
     }
 
     public <T> ReportNode buildModificationReport(T oldValue, T newValue, String fieldName) {
-        return buildModificationReport(oldValue, newValue, fieldName, 1, TypedValue.INFO_SEVERITY);
-    }
-
-    public <T> ReportNode buildModificationReport(T oldValue, T newValue, String fieldName, int indentationLevel) {
-        return buildModificationReport(oldValue, newValue, fieldName, indentationLevel, TypedValue.INFO_SEVERITY);
+        return buildModificationReport(oldValue, newValue, fieldName, TypedValue.INFO_SEVERITY);
     }
 
-    public static <T> ReportNode buildModificationReport(T oldValue, T newValue, String fieldName, int indentationLevel, TypedValue severity) {
+    public static <T> ReportNode buildModificationReport(T oldValue, T newValue, String fieldName, TypedValue severity) {
         final String oldValueString = (oldValue == null || oldValue instanceof Double oldDouble && Double.isNaN(oldDouble))
                 ? NO_VALUE : oldValue.toString();
         final String newValueString = (newValue == null || newValue instanceof Double newDouble && Double.isNaN(newDouble))
                 ? NO_VALUE : newValue.toString();
-        final String indentation = "\t".repeat(indentationLevel);
```

Comment:
we lose indentation ?

Reply:
Yes, all indentations will be removed. This change is approved by the PO.